### PR TITLE
Add Prod and Preprod pipelines with example secrets.

### PR DESCRIPTION
**Pull request by rjweeks**
# Motivation and Context
Add response-operations-social-ui pipeline with preprod and prod deploy jobs.

# What has changed
Added new response-operations-social-ui pipeline.
Added task to run python unit tests in a docker container with the make test command
Added example secrets file for new pipeline

# How to test?
This pipeline has been flown and successfully deployed r-ops-social to preprod and prod.

# Links
https://trello.com/c/8cG7FHHn/197-sus001-int-access-social-survey-13

### CODE REVIEW

**Comment by AdamHawtin** [comment]
[SUS001: INT Access Social Survey (13)](https://trello.com/c/8cG7FHHn/197-sus001-int-access-social-survey-13)

**Review by davidmort — APPROVED** [review]

File: pipelines/response-operations-social-ui.yml [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+  type: git
+  source:
+    uri: https://github.com/ONSdigital/response-operations-social-ui.git
+    branch: master
+
+- name: ras-deploy
+  type: git
+  source:
+    uri: https://github.com/ONSdigital/ras-deploy.git
+    branch: response-ops-social-prod
```
Comment (liamtoozer) [inline pipelines/response-operations-social-ui.yml]: Remember to change this to `branch: master` before merging this PR 👍

File: pipelines/response-operations-social-ui.yml [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+
+jobs:
+- name: response-operations-social-ui-preprod-deploy
+  serial: true
+  plan:
+  - get: response-operations-social-ui-source
+    trigger: true
+  - get: ras-deploy
+  - task: run-unit-tests
+    file: ras-deploy/tasks/python-unit-tests.yml
```
Comment (liamtoozer) [inline pipelines/response-operations-social-ui.yml:80]: ~~Not sure what the agreed approach is for the team, but we could maybe put this in the r-ops-social repo instead? That way, ras-deploy as a resource would no longer be needed in here. This has been done in RH https://github.com/ONSdigital/respondent-home-ui/tree/master/ci. Not sure what's for the best though...~~

Sorry ignore that, I can see they're for generic use!

**Review by liamtoozer — APPROVED** [review]
LGTM!

File: pipelines/response-operations-social-ui.yml [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+        SAMPLE_PASSWORD: ((preprod_security_user_password))
+        SECURITY_USER_NAME: ((preprod_security_user_name))
+        SECURITY_USER_PASSWORD: ((preprod_security_user_password))
+        REDIS_SERVICE: res-ops-redis
+  - put: map-route-vpn-domain
+    resource: cf-cli-resource-preprod
+    params:
+      command: map-route
+      app_name: response-operations-social-ui-preprod
+      domain: sdc-social-preprod.rmdev.onsdigital.uk
```
Comment (liamtoozer) [inline pipelines/response-operations-social-ui.yml]: Should these domains be secret like above?

Comment (jcox-dev) [reply pipelines/response-operations-social-ui.yml]: +1 (and the prod domain below)

**Pull request merged**